djstripe/models.py

```diff
@@ -21,6 +22,7 @@
 import stripe
 
 from . import settings as djstripe_settings
+from djstripe.managers import StripeObjectManager
 from .exceptions import SubscriptionCancellationFailure, SubscriptionUpdateFailure
 from .managers import CustomerManager, ChargeManager, TransferManager
```

Comment:
My IDE does these imports automatically, not always the prettiest! Sorry, will fix and be more vigilant